tutorials/multi-tenant-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+.. _multi_tenant_tutorial:
+.. highlight:: sql
+
+Multi-tenant data model
+########################
+
+In this tutorial, we will use a sample ad analytics dataset to demonstrate how you can       
+use Citus to power your multi-tenant application.                             
+
+.. note::
+                                                                                             
+    This tutorial assumes that you already have Citus installed and running. If you don't have Citus running,
+    you can provision a cluster using `Citus Cloud <https://console.citusdata.com>`_ or setup Citus locally
+    using :ref:`single_machine_docker`.
```

Comment:
@samay-sharma -- I opened #292 to cover two of the three bullet points in this comment.

We still ask the user to shut down Docker in our docs. If we point them to the Docker page, it may introduce a minor flow issue. Are we worried about that at all -- enough to capture it some place?